include/circt/Dialect/RTL/RTLCombinatorial.td

```diff
@@ -187,21 +187,9 @@ def OrROp  : UnaryI1ReductionRTLOp<"orr">;
 def XorROp : UnaryI1ReductionRTLOp<"xorr">;
 
 //===----------------------------------------------------------------------===//
-// Other Operations
+// Width-modifying unary operations on ints.
```

Comment:
I'd drop "on ints", also you might just want to call this "Bit extraction" since there is only one thing in it so far.